### ISSUE TITLE
Add "check for new version" to osvr_server

### ISSUE THREAD

**Issue by mars979** [OSVR/OSVR-Core#272]
Have osvr server periodically (or at least upon launch) check if newer version of the server is available.

**Comment by d235j** [comment]
Should this be part of the osvr_server binary, or should we plan a GUI "OSVR Manager" app of sorts that handles this among other config-related things?

**Comment by godbyk** [comment]
I think it should be part of a manager/configurator app to come later. I envision osvr_server evolving into a combination of a daemon (Linux, Mac OS X) or service (Windows) and control/manager application (which will likely have a GUI as well as command-line interface).

**Comment by rpavlik** [comment]
Yeah - As @d235j has likely discovered, the current osvr_server app is barely more than a command-line stub/wrapper around the osvrServer library where the actual stuff gets done.

I think there's already a bug open about an installer/updater, fwiw.
